lib/rules/space-in-parens.js

```diff
@@ -184,9 +188,9 @@ module.exports = function(context) {
 
     return {
         "Program": function checkParenSpaces(node) {
-            var tokens = node.tokens;
-            var prevToken, nextToken;
+            var tokens, prevToken, nextToken;
             exceptions = getExceptions();
+            tokens = sourceCode.tokensAndComments;
```

Comment:
What meant was, it would still be calculated once but just to get the data wewill have a function like getTokenComment or something.